generics/services/users.js

```diff
@@ -16,13 +16,15 @@ const profile = function ( token,userId = "" ) {
             let url = userServiceUrl + CONSTANTS.endpoints.USER_READ_V5;
     
             if( userId !== "" ) {
-                url = url + "/" + userId + "?"  + "fields=organisations,roles,locations,declarations,externalIds"
+                url = url + "/" + userId 
             }
 
+
             const options = {
                 headers : {
-                    "content-type": "application/json",
-                    "x-authenticated-user-token" : token
+                    // "content-type": "application/json",
```

Comment:
@Prajwal17Tunerlabs Please check if we require content-type

Reply:
i have un-commented it and also it is tested